Jenkinsfile

```diff
@@ -0,0 +1,183 @@
+import groovy.json.JsonOutput
+
+/** Map of Tox environments and associated capabilities */
+def environments = [
+  desktop: [
+    browserName: 'Firefox',
+    version: '47.0',
+    platform: 'Windows 7'
+  ],
+  mobile: [
+    browserName: 'Browser',
+    platformName: 'Android',
+    platformVersion: '5.0',
+    deviceName: 'Android Emulator',
+    appiumVersion: '1.5.3'
+  ]
+]
```

Comment:
@davehunt how is this declaration associated with the tox envlist?  is tox running through the various envs called:  "desktop" and "mobile" (which are here defined within the Jenkinsfile)?